suite-common/firmware/src/firmwareThunks.ts

```diff
@@ -165,6 +165,8 @@ export const firmwareUpdate = createThunk<
                         errorMessage: firmwareUpdateResponse.payload.checkError,
                     }),
                 );
+            } else if (!binary && !versionCheck) {
+                // TODO: log to sentry
```

Comment:
I can do it.

Reply:
[Example log in Sentry 2](https://satoshilabs.sentry.io/issues/6270089317/events/646858e35eb74eefbbae0fa9c8a41bb5/)
Mocked by changing `installedVersion` [here](https://github.com/trezor/trezor-suite/blob/8a8a119931982a99e042235a587b7fcfe840a14c/packages/connect/src/core/onCallFirmwareUpdate.ts#L547).